Vagrant.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
```

Comment:
Could you update Chef cookbooks instead? I prefer to use a tool like Chef(or a Python alternative) instead of a handwritten bash script.